public/icons/beta.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 102 54" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
+    <g transform="matrix(1,0,0,1,-7234.57,-38681.8)">
+        <g id="typo">
+        </g>
+        <g id="typo1" serif:id="typo">
+        </g>
+        <g id="typo2" serif:id="typo">
+        </g>
+        <g id="typo3" serif:id="typo">
+        </g>
+        <g id="_--logo-resana-1" serif:id="• logo resana 1">
+        </g>
+        <g id="_--logo-resana-2" serif:id="• logo resana 2">
+        </g>
+        <g id="_--frise" serif:id="• frise">
+        </g>
+        <g id="beta" transform="matrix(1,0,0,1,-94.448,-50.8646)">
+            <rect x="7329.02" y="38732.6" width="101.932" height="53.623" style="fill:none;"/>
+            <g transform="matrix(1.00045,0,0,1.00045,-401.862,-184.022)">
+                <g transform="matrix(2.48487,0,0,2.48487,7718.94,38902.1)">
+                    <path d="M40.094,2L8,2C6.9,2 6,2.9 6,4L6,16C6,17.1 6.9,18 8,18L40.094,18C41.194,18 42.094,17.1 42.094,16L42.094,4C42.094,2.9 41.194,2 40.094,2Z" style="fill:rgb(0,0,145);fill-rule:nonzero;"/>
+                </g>
+                <g transform="matrix(1,0,0,1.29806,-15.3425,-11613.4)">
+                    <path d="M7838.88,38942.6L7838.88,38950.7L7829.7,38950.7L7829.7,38942.6L7838.88,38942.6ZM7758.38,38920L7758.38,38928L7749.19,38928L7749.19,38920L7758.38,38920Z" style="fill:rgb(0,0,145);"/>
+                </g>
+            </g>
+            <g transform="matrix(0.141442,0,0,0.141442,6293.27,33289.2)">
+                <g transform="matrix(235.512,0,0,235.512,7957.75,38763.6)">
+                </g>
+                <text x="7406.41px" y="38763.6px" style="font-family:'Marianne-Bold', 'Marianne';font-weight:700;font-size:235.512px;fill:white;">B<tspan x="7539.95px 7664.3px 7782.06px " y="38763.6px 38763.6px 38763.6px ">ETA</tspan></text>
```

Comment:
I'm pretty sure this doesn't work for people not having the font locally. Tried this on browserstack for example and it's rendered as basic serif. SVGs don't go check the at-font-face CSS rules if I'm not mistaken…

I'd rather render "BETA" as classic html/css since the svg just outputs a text tag anyway :)